### PR TITLE
Use refs for long variable names

### PR DESCRIPTION
**Pull request by anastasia-popova**
## Purpose 

Cleans up long variable names found in experiments/AMIP and src

closes [#234](https://github.com/CliMA/ClimaCoupler.jl/issues/234)

## Content

-Found long variable names in experiments/AMIP and src
-Locally renamed with refs
-Verified that there is no change through bit wise comparison

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

File: experiments/AMIP/components/atmosphere/climaatmos_init.jl [hunk 1]
```diff
@@ -64,6 +64,7 @@ end
 
 function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
 
+
```
Comment (juliasloan25) [inline experiments/AMIP/components/atmosphere/climaatmos_init.jl:67]: can you remove these extra lines of whitespace please?

File: experiments/AMIP/components/atmosphere/climaatmos_init.jl [hunk 2]
```diff
@@ -76,16 +77,27 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
         @warn("Running with ρe_int in coupled mode is not tested yet.", maxlog = 1)
     end
 
+    # define variables
```
Comment (juliasloan25) [inline experiments/AMIP/components/atmosphere/climaatmos_init.jl:80]: maybe a more descriptive comment would be helpful here - something like "define shorter refs for long variable names to increase readability", just to emphasize that this doesn't affect the behavior

**Review by juliasloan25 — COMMENTED** [review]
This looks really good, thank you Anastasia! I just left a couple of suggestions. After you address those, you can re-request my review and it should be good to go :)

File: experiments/AMIP/components/atmosphere/climaatmos_init.jl [hunk 3]
```diff
@@ -76,16 +77,27 @@ function atmos_init(::Type{FT}, atmos_config_dict::Dict) where {FT}
...
-    integrator.p.precipitation.col_integrated_rain .= FT(0)
-    integrator.p.precipitation.col_integrated_snow .= FT(0)
-    integrator.p.precipitation.ᶜS_ρq_tot .= FT(0)
-    integrator.p.precipitation.ᶜ3d_rain .= FT(0)
-    integrator.p.precipitation.ᶜ3d_snow .= FT(0)
+    @. ρ_flux_h_tot = Geometry.Covariant3Vector(FT(0.0))
+    @. ρ_flux_q_tot = Geometry.Covariant3Vector(FT(0.0))
+    @. ᶠradiation_flux = Geometry.WVector(FT(0))
+    ρ_flux_uₕ.components .= Ref(SMatrix{1, 2}([FT(0), FT(0)]))
+    col_integrated_rain .= FT(0)
```
Comment (LenkaNovak) [inline experiments/AMIP/components/atmosphere/climaatmos_init.jl:96]: You could use the `@.` broadcasting macro everywhere, but that's just aesthetics. :)

File: experiments/AMIP/components/atmosphere/climaatmos_init.jl [hunk 4]
```diff
@@ -307,22 +319,10 @@ function get_field(atmos_sim::ClimaAtmosSimulation, ::Val{:F_radiative_TOA})
...
-        )
-        SWd_TOA = Fields.level(
-            CA.RRTMGPI.array2field(FT.(atmos_sim.integrator.p.radiation.radiation_model.face_sw_flux_dn), face_space),
-            nz_faces - half,
-        )
-        SWu_TOA = Fields.level(
-            CA.RRTMGPI.array2field(FT.(atmos_sim.integrator.p.radiation.radiation_model.face_sw_flux_up), face_space),
-            nz_faces - half,
-        )
+        LWd_TOA = Fields.level(CA.RRTMGPI.array2field(FT.(radiation.face_lw_flux_dn), face_space), nz_faces - half)
```
Comment (LenkaNovak) [inline experiments/AMIP/components/atmosphere/climaatmos_init.jl:322]: You could in fact unpack the individual variables, without allocating, as follows:
`(; face_lw_flux_dn, face_lw_flux_up, face_sw_flux_dn, face_sw_flux_up) = atmos_sim.integrator.p.radiation.radiation_model`

File: experiments/AMIP/components/land/bucket_utils.jl [hunk 5]
```diff
@@ -79,9 +81,9 @@ reinit!(sim::BucketSimulation) = reinit!(sim.integrator)
 # extensions required by FluxCalculator (partitioned fluxes)
 function update_turbulent_fluxes_point!(sim::BucketSimulation, fields::NamedTuple, colidx::Fields.ColumnIndex)
     (; F_turb_energy, F_turb_moisture) = fields
-    sim.integrator.p.bucket.turbulent_fluxes.shf[colidx] .= F_turb_energy
-    sim.integrator.p.bucket.turbulent_fluxes.vapor_flux[colidx] .=
-        F_turb_moisture ./ LP.ρ_cloud_liq(sim.model.parameters.earth_param_set)
+    turbulent_fluxes = sim.integrator.p.bucket.turbulent_fluxes
+    turbulent_fluxes.shf[colidx] .= F_turb_energy
+    turbulent_fluxes.vapor_flux[colidx] .= F_turb_moisture ./ LP.ρ_cloud_liq(sim.model.parameters.earth_param_set)
     return nothing
```
Comment (LenkaNovak) [inline experiments/AMIP/components/land/bucket_utils.jl:87]: We could also define `earth_params = sim.model.parameters.earth_param_set`

**Review by LenkaNovak — COMMENTED** [review]
This looks great, thank you, @anastasia-popova! I only had a few minor comments.

**Review by juliasloan25 — APPROVED** [review]
Looks good to me! You can rebase on main and then if Lenka approves you can merge :)

**Review by juliasloan25 — APPROVED** [review]

**Review by LenkaNovak — APPROVED** [review]

**Pull request merged**